src/App.css

```diff
@@ -1,3 +1,9 @@
 .App {
   font-family: 'Roboto', sans-serif;
 }
+
+#gauge{
+  position: absolute;
+  right: 10vh;
+  top: 10vh;
+}
```

Comment:
files should always end with one blank newline

Reply:
Thanks, Ill keep that in mind going forward.